### PR TITLE
#214 - add docs for space-separated scope event syntax

### LINKED ISSUE #214: Add documentation for the space-separated even syntax

### PR DESCRIPTION
**Pull request by mickmcgrath13**
resolves #214

### CODE REVIEW

File: docs/event.md [hunk 1]
```diff
@@ -43,6 +43,29 @@ is fired. The following key values are also supported:
...
 
+@signature `(SCOPE_KEY_LOOKUP SCOPE_EVENT)='CALL_EXPRESSION'`
+
+Listens to an event in the [can-stache/expressions/key-lookup specified scope] and calls the [can-stache/expressions/call] when that event occurs.
+
+```
+<my-component (. someevent)="doSomething()"/>
+```
+
+@param {can-stache/expressions/key-lookup} SCOPE_KEY_LOOKUP A key to specify what scope the SCOPE_EVENT is in.
```
Comment (justinbmeyer) [inline docs/event.md]: not really the "scope"  ... there's only one scope.  It's really what object's (or event emitters) events should be bound to.

File: docs/event.md [hunk 2]
```diff
@@ -43,6 +43,29 @@ is fired. The following key values are also supported:
  - `%arguments` - The arguments passed when the event was dispatched/triggered.
 
 
+@signature `(SCOPE_KEY_LOOKUP SCOPE_EVENT)='CALL_EXPRESSION'`
+
+Listens to an event in the [can-stache/expressions/key-lookup specified scope] and calls the [can-stache/expressions/call] when that event occurs.
+
+```
+<my-component (. someevent)="doSomething()"/>
```
Comment (justinbmeyer) [inline docs/event.md]: can we use something other than `.`?  Maybe show a bit more of an example to make the context more clear?

File: docs/event.md [hunk 3]
```diff
@@ -43,6 +43,29 @@ is fired. The following key values are also supported:
  - `%arguments` - The arguments passed when the event was dispatched/triggered.
 
 
+@signature `(SCOPE_KEY_LOOKUP SCOPE_EVENT)='CALL_EXPRESSION'`
```
Comment (justinbmeyer) [inline docs/event.md]: Maybe this should just be `KEY_LOOKUP` and `EVENT`

File: docs/event.md [hunk 4]
```diff
@@ -43,6 +43,29 @@ is fired. The following key values are also supported:
...
+
+Listens to an event in the [can-stache/expressions/key-lookup specified scope] and calls the [can-stache/expressions/call] when that event occurs.
+
+```
+<my-component (. someevent)="doSomething()"/>
+```
+
+@param {can-stache/expressions/key-lookup} SCOPE_KEY_LOOKUP A key to specify what scope the SCOPE_EVENT is in.
+
+@param {String} SCOPE_EVENT An event in the specified scope.
```
Comment (justinbmeyer) [inline docs/event.md]: The name of the event on an event emitter specified by `KEY_LOOKUP`.

**Review by justinbmeyer — CHANGES_REQUESTED** [review]

File: docs/event.md (hunk 3 above)
Comment (mickmcgrath13) [reply docs/event.md]: 👍

File: docs/event.md (hunk 2 above)
Comment (mickmcgrath13) [reply docs/event.md]: 👍

File: docs/event.md (hunk 1 above)
Comment (mickmcgrath13) [reply docs/event.md]: Would 'context' be better, or is that something different, too?

"A key to specify the context for the SCOPE_EVENT."

File: docs/event.md [hunk 5]
```diff
@@ -43,6 +43,43 @@ is fired. The following key values are also supported:
...
+```
+In this example, `activityrecorder` exists on the current scope and is defined as:
+```
+var ActivityRecorder = DefineMap.extend({
+	recordClick: function(){
+		this.dispatch("clickrecorded");
+	}
+});
+
+var scope = new DefineMap({
```
Comment (justinbmeyer) [inline docs/event.md]: var context

File: docs/event.md [hunk 6]
```diff
@@ -43,6 +43,43 @@ is fired. The following key values are also supported:
...
+
+Listens to an event in the [can-stache/expressions/key-lookup specified scope] and calls the [can-stache/expressions/call] when that event occurs.
+
+```
+<div 
+  ($click)="./activityrecorder.recordClick()"
+  (./activityrecorder clickrecorded)="doSomething()"
+/>
+```
+In this example, `activityrecorder` exists on the current scope and is defined as:
```
Comment (justinbmeyer) [inline docs/event.md]: is on the current context

File: docs/event.md [hunk 7]
```diff
@@ -43,6 +43,43 @@ is fired. The following key values are also supported:
...
+		this.dispatch("clickrecorded");
+	}
+});
+
+var scope = new DefineMap({
+	activityrecorder: new ActivityRecorder()
+})
+```
+
+@param {can-stache/expressions/key-lookup} KEY_LOOKUP A key to specify the context for the `EVENT`.
```
Comment (justinbmeyer) [inline docs/event.md]: A key to specify the context for the `EVENT` binding. The object specified by `key` will have its event named `EVENT` bound.

**Review by justinbmeyer — CHANGES_REQUESTED** [review]

**Comment by justinbmeyer** [comment]
If we've never added docs for this, we probably shouldn't.  Instead we should make sure the new binding syntax `on:something:by:somethingElse` is documented and works.

**Comment by phillipskevin** [comment]
`on:foo:by:obj` is documented and works. Docs will be released in 3.10.0.

**Pull request closed without merging**